projects/observability/src/shared/graphql/model/schema/filter/id/graphql-id-filter.ts

```diff
@@ -4,12 +4,16 @@ import { GraphQlFilter, GraphQlFilterType, GraphQlOperatorType } from '../graphq
 export class GraphQlIdFilter implements GraphQlFilter {
   public readonly key: string = 'id';
 
-  public constructor(public readonly id: string, public readonly idScope: string) {}
+  public constructor(
+    public readonly id: string,
+    public readonly idScope: string,
+    public readonly operator: GraphQlOperatorType = GraphQlOperatorType.Equals
+  ) {}
 
   public asArgumentObjects(): [GraphQlIdFilterArgument] {
     return [
       {
-        operator: new GraphQlEnumArgument(GraphQlOperatorType.Equals),
+        operator: new GraphQlEnumArgument(this.operator),
```

Comment:
Any reason why we couldn't use a GraphQlFieldFilter if we want to use a different operator?

Reply:
Do you mean use a `GraphQlFieldFilter` for other operators? We can, at least in the case I need this for. But using the ID filter is a little more semantically correct and helps readers better understand intent.

Not sure if there is more under the hood that this also helps, maybe @aaron-steinfeld can comment.